react-frontend-app/src/components/MainNavBar.js

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
 import '../style/MainNavBar.css';
 
-// TO-DO: AUSTIN(?)
-
-class MainNavBar extends React.Component {
-    constructor(props) {
-        super(props)
-        // You should store stateful data here
-        this.state = {}
-    }
-
-    render () {
-        return (
-            <div>
-                <p>I am the Zest Navigation Bar (purely cosmetic until we have multiple pages). I'm very sad b/c I have no code here yet</p>
-            </div>
-        )
-    }
+function MainNavBar() {
```

Comment:
You really love functional React hahahah

Reply:
;)